### DISCUSSION TITLE
Cap'n Proto mailing list has moved to GitHub discussions

### DISCUSSION THREAD

**Discussion by kentonv** [capnproto/capnproto#1848] [Announcements]
As many of you may have noticed, the Cap'n Proto mailing list has had a spam problem lately. I don't know what changed. We never had a problem for 10+ years, and then suddenly, Google couldn't keep the spammers out anymore.

Seeing as how Google Groups appears to be a dead product kept around only because Google uses it internally, it's probably time to give up on it. To that end, going forward we will use GitHub discussions instead. Please update your notification settings accordingly.

(Gory details: The setting I've always used in Google Groups is to require moderation for messages from non-members of the group. But if you *joined* the group, you could post right away. I guess the spam bots have suddenly learned how to join the group before posting. I could change things so all new members start out moderated. But in my experience, people are incredibly confused by moderation, as Google Groups doesn't seem to give them any indication of why their message didn't post. Plus, I'm often very slow to approve messages. Increasing moderation seems like a loss.)

**Comment by griff** [comment]
The website still points to the Google Groups list.
